docs/concepts/models/sql_models.md

```diff
@@ -1,37 +1,40 @@
 # SQL models
 
-SQL models are the main types of models used by SQLMesh. SQL models consist of three sections:
+SQL models are the main type of models used by SQLMesh. These models can be defined using either SQL or Python that generates SQL.
 
-* `MODEL` DDL
-* Optional sql statements
-* The model's `SELECT` statement
+## SQL-based definition
 
-SQL models are designed to look and feel like you're simply using SQL, but they can be customized for advanced use cases.
+The SQL-based definition of SQL models is the most common one, and consists of the following sections:
 
-## Definition
-To create an example SQL model, add a file named `my_model.sql` into the `models/` directory (or a subdirectory of `models/`) within your SQLMesh project.
+* The `MODEL` DDL
+* Optional pre-statements
+* A single query
+* Optional post-statements
 
-Although the name of the file doesn't matter, it is recommended to name it something that includes the word "model". Each file can only have one model defined within it.
+These models are designed to look and feel like you're simply using SQL, but they can be customized for advanced use cases.
+
+To create a SQL-based model, add a new file with the `.sql` suffix into the `models/` directory (or a subdirectory of `models/`) within your SQLMesh project. Although the name of the file doesn't matter, it is recommended to use a name that preserves the model's name. Each file can only have one model defined within it.
 
 ### Example
+
 ```sql linenums="1"
--- The Model DDL where you specify model metadata and configuration information.
+-- This is the MODEL DDL, where you specify model metadata and configuration information.
 MODEL (
   name db.customers,
   kind FULL,
 );
 
 /*
-  Optional SQL statements to run before the model's `SELECT` statement.
-  You should NOT do things that cause side effects that could
-  error out when multiple queries run, such as creating physical tables.
+  Optional pre-statements that will run before the model's query.
+  You should NOT do things that cause side effects that could error out when
+  executed concurrently with other statements, such as creating physical tables.
 */
 CACHE TABLE countries AS SELECT * FROM raw.countries;
 
 /*
-  This is the main SQL statement that comprises your model.
-  Although it is not required, it is best practice to specify explicit types
-  for all columns in the final SELECT statement.
+  This is the single query that defines the model's logic.
+  Although it is not required, it is considered best practice to explicitly
+  specify the type for each one of the model's columns through casting.
```

Comment:
"...through casting. SQLMesh supports the casting format `<column name>::<data type>` in models of any SQL dialect."